### PR TITLE
fix: select filter date (again)

### PR DESCRIPTION
**Pull request by rephus**
Closes: #3583
### Description:

Dropping some date conversions, converting to string to avoid timezone issues 

Before:
[Screencast from 26-10-22 09:06:13.webm](https://user-images.githubusercontent.com/1983672/197959008-5f1ec369-cc36-4123-912a-dc06a58f0602.webm)

After:

[Screencast from 26-10-22 09:13:28.webm](https://user-images.githubusercontent.com/1983672/197959154-2ffc9f18-cab9-4ecf-9162-7d3e82e80993.webm)

### CODE REVIEW

**Comment by rephus** [comment]
Soon... https://github.com/lightdash/lightdash/actions/runs/3327647471/jobs/5502710205

**Comment by rephus** [comment]
Also tested on Japan 
[Screencast from 26-10-22 20:11:48.webm](https://user-images.githubusercontent.com/1983672/198011935-efaba789-2e2c-4f6b-b487-98a0290c7e94.webm)

File: packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx [hunk 1]
```diff
@@ -131,22 +131,25 @@ const DateFilterInputs: FC<FilterInputsProps<DateFilterRule>> = (props) => {
                     className={disabled ? 'disabled-filter' : ''}
                     disabled={disabled}
                     fill
-                    showTimezoneSelect={true}
                     value={
                         filterRule.values?.[0]
-                            ? new Date(filterRule.values?.[0]).toString()
+                            ? formatDate(
+                                  filterRule.values?.[0],
+                                  undefined,
```
Comment (ZeRego) [inline packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx:138]: A problem when filtering from results
filter timestamp

Perhaps we need to pass the time interval when we call formatDate.

Comment (rephus) [reply packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx:138]: [Screencast from 26-10-22 14:20:12.webm](https://user-images.githubusercontent.com/1983672/198025421-ae3a4739-6007-478a-8852-b2d55d5efd44.webm)

**Review by ZeRego — APPROVED** [review]

**Pull request merged**